internal/kafka/internal/api/public/api/openapi.yaml

```diff
@@ -1800,6 +1800,8 @@ components:
           type: string
         instance_type:
           type: string
+        display_name:
+          type: string
```

Comment:
```suggestion
        instance_type_name:
          type: string
```
possible renaming suggestion to avoid confusion between the relationship of `name` vs `display_name` in the API model.

Reply:
@pb82 what do you think of this renaming proposal?

Reply:
@pb82 - any opinion?

Reply:
But do `name` and `display_name` appear in the same APIs?

Reply:
Yes. 
When you do a `GET kafkas` or `GET kafkas by ID`  name and display_name will appear on the returned Kafkas.

Reply:
I see. Can we make it so that `instance_type_name` is what we return from the endpoint, while in app-interface it's still `display_name`?

Reply:
`I see. Can we make it so that instance_type_name is what we return from the endpoint, while in app-interface it's still display_name?` - yes

Reply:
changed to `instance_type_name`

Reply:
Thanks.